tools.html

```diff
@@ -2,7 +2,7 @@
 layout: general
 highlight_section: tools
 title: Get tools
-description: Prepare your environment with DevAssistant. Collaborate with Docker and Vagrant.
+description: Prepare your environment with DevAssistant. Collaborate with Docker, Vagrant and Virt-builder.
```

Comment:
Yes it is not.  Actually, `virt-builder` should have been included in the first sentence: "Prepare your environment with DevAssistant, and Virt-builder", since its role is in preparing an environment (VM+associated tooling) for development.

Do you want me to respin and update it, or can you update it during commit?